Add nopStation ecommerce as technology

kazirahiv opened the pull request:
NopStation builts custom ecommerce solutions with custom integrations and custom built plugins based on tailored requirements on top of NopCommerce. Adding entry to detect sites built with nopCommerce and nopCommerce plugins

nurbek91 commented:
Hey mate, thanks for PR.
I'm not sure if nopStation can be added as a separate technology for ecommerce
examples:
https://nrbbazaar.com/
https://purplebybanu.com/
https://gasgasshopping.at/
https://motoshop24.ch/

On all the examples that I have tested it is either plugins or themes for nopCommerce
Probably it will be possible to add technology under category 19

nurbek91 commented on src/technologies/n.json:
```diff
@@ -845,6 +845,20 @@
+    "icon": "nopStation.png",
+    "implies": "Microsoft ASP.NET",
+    "meta": {
+      "generator": "^nopStation$"
```
Do you have examples of sites with this fingerprint?

kazirahiv replied:
removed the meta tag

nurbek91 commented on src/technologies/n.json:
```diff
@@ -845,6 +845,20 @@
+      6
+    ],
+    "description": "One stop e-commerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of NopCommerce",
+    "html": "<a[^>]+nop-station",
```
It is not advisable to use html now
it can be replaced with dom

kazirahiv replied:
Replaced with dom

nurbek91 commented on src/technologies/n.json:
```diff
@@ -845,6 +845,20 @@
+    "meta": {
+      "generator": "^nopStation$"
+    },
+    "oss": true,
```
I think this field is invalid

kazirahiv replied:
Removed the oss field

nurbek91 commented on src/technologies/n.json:
```diff
@@ -845,6 +845,20 @@
+    "cats": [
+      6
+    ],
+    "description": "One stop e-commerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of NopCommerce",
```
nopStation is a one stop ecommerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of nopCommerce.

nurbek91 reviewed (changes requested):
.

nurbek91 commented:
![nopStation](https://user-images.githubusercontent.com/3456008/173186291-fa18e69c-7050-4f85-aee0-de67e7a83579.png)
32x32

kazirahiv commented:
@nurbek91 Hi, Made the changes, Added a new commit. Can you please check now ?

kazirahiv commented:
> 

There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.

nurbek91 commented:
> > 
> 
> There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.

Hi, for 6 we try to define always one technology.
So we need to add the attribute:
`"exclude": "nopCommerce"`

But I'm not sure about this.

kazirahiv commented:
> > > 
> > 
> > 
> > There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.
> 
> Hi, for 6 we try to define always one technology. So we need to add the attribute: `"exclude": "nopCommerce"`
> 
> But I'm not sure about this.

Added the excludes property.

nurbek91 commented:
> > > > 
> > > 
> > > 
> > > There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.
> > 
> > 
> > Hi, for 6 we try to define always one technology. So we need to add the attribute: `"exclude": "nopCommerce"`
> > But I'm not sure about this.
> 
> Added the excludes property.

But I'm not sure about this. It will probably still be correct to specify category 19 and add requires `"requires": "nopCommerce"`

nurbek91 commented:
Exactly how it will be right most likely can  only  @AliasIO  tell

kazirahiv commented:
no problem, I made the changes to cat 19 and requires nopCommerce @nurbek91

AliasIO commented:
Thank you!

rockeynebhwani commented:
@AliasIO @kazirahiv - We are missing 'nopStation.png'.
